packages/components/src/input-control/index.tsx

```diff
@@ -8,7 +8,7 @@ import type { ForwardedRef } from 'react';
  * WordPress dependencies
  */
 import { useInstanceId } from '@wordpress/compose';
-import { useState, forwardRef } from '@wordpress/element';
+import { useState, forwardRef } from 'react';
```

Comment:
This should be @wordpress/element

Reply:
Reverted to `@wordpress/element` import. 

The context here is source code ([see notes on codebase consistency](https://github.com/WordPress/gutenberg/pull/59517#:~:text=we%20use%20react-,For%20code%2C%20we%27re%20still%20using%20%40wordpress/element,-So%20let%27s%20just))

Reply:
This is still using "react" though? Unless GitHub is not showing me the latest changes :)